Makefile

```diff
@@ -176,6 +177,7 @@ run: manifests generate fmt vet ## Run a controller from your host.
 PLATFORMS ?= linux/arm64,linux/amd64,linux/s390x,linux/ppc64le
 .PHONY: docker-buildx
 docker-buildx: ## Build and push docker image for the manager for cross-platform support
+	export GOPROXY=$(GOPROXY)
```

Comment:
It's better not to export GOPROXY If GOPROXY=""

Reply:
I opened https://github.com/InftyAI/llmaz/pull/330 to fix this. Is that better?